integration-tests/src/test/java/tachyon/client/keyvalue/KeyValuePartitionIntegrationTest.java

```diff
@@ -76,4 +78,62 @@ public void readerWriterTest() throws IOException, TachyonException {
     Assert.assertArrayEquals(VALUE2, mKeyValuePartitionReader.get(KEY2));
     Assert.assertNull(mKeyValuePartitionReader.get("NoSuchKey".getBytes()));
   }
+
+  /**
+   * Tests that {@link KeyValuePartitionReader#size()} is correct when a new reader is created.
+   *
+   * @throws Exception when any exception happens
```

Comment:
for test cases, we don't need to add exceptions in javadoc (nobody will need this info any way)